README.md

```diff
@@ -34,26 +22,6 @@ Then you can interact with hubot by typing `hubot help`.
     hubot help - Displays all of the help commands that hubot knows about.
     ...
 
-### Configuration
-
-A few scripts (including some installed by default) require environment
-variables to be set as a simple form of configuration.
-
-Each script should have a commented header which contains a "Configuration"
-section that explains which values it requires to be placed in which variable.
-When you have lots of scripts installed this process can be quite labour
-intensive. The following shell command can be used as a stop gap until an
-easier way to do this has been implemented.
-
-    grep -o 'hubot-[a-z0-9_-]\+' external-scripts.json | \
-      xargs -n1 -I {} sh -c 'sed -n "/^# Configuration/,/^#$/ s/^/{} /p" \
-          $(find node_modules/{}/ -name "*.coffee")' | \
-        awk -F '#' '{ printf "%-25s %s\n", $1, $2 }'
-
-How to set environment variables will be specific to your operating system.
-Rather than recreate the various methods and best practices in achieving this,
-it's suggested that you search for a dedicated guide focused on your OS.
-
 ### Scripting
```

Comment:
It doesn't hurt, but I will tweak a little. Our internal hubot still links to this scripting reference.